Common/MathUtils/include/MathUtils/Trans3D.h

```diff
@@ -0,0 +1,75 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See https://alice-o2.web.cern.ch/ for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#ifndef ALICEO2_BASE_TRANS3D_H_
+#define ALICEO2_BASE_TRANS3D_H_
+
+#include "MathUtils/Cartesian3D.h"
```

Comment:
I am wondering whether we could put the Trans3D class inside the Cartesian3D header? Then its name would be justified...